frontend.yml

```diff
@@ -23,6 +23,10 @@ objects:
         paths:
           - /
           - /config/chrome
+      akamaiCacheBustPaths:
+        - /config/chrome/fed-modules.json
```

Comment:
Just curious. How is `/beta` path handled? Is it dynamically prefixed somewhere or do we have to define it?

Reply:
Yup, beta and preview are added based on the frontened environment